samples/dlp_delete_job.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(projectId = 'YOUR_PROJECT_ID', dlpJob = 'YOUR_JOB_NAME') {
+  // [START dlp_delete_job]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'YOUR_PROJECT_ID';
+  // const dlpJob = 'X-abcd-1234';
+
+  // Imports the Google Cloud Data Loss Prevention library
+  const dlp = require('@google-cloud/dlp');
```

Comment:
This should be 
`const {DlpServiceClient} = require('@google-cloud/dlp');` 
and then 
`const client = new DlpServiceClient();`

Reply:
Done, though instead of `client` I did the camelCased version of `DlpServiceClient` so that the sample autogen tooling can easily handle the case where a sample needs to use two different services clients from the same library (example: Monitoring API).